### ISSUE TITLE
June 2020 Update Suggestions

### ISSUE THREAD

**Issue by timahenning** [microsoft/MCW-Intelligent-analytics#48]

**Comment by timahenning** [comment]
- I have run through the lab about 6 to 7 times.  My best time is about 5.5 hours.   I know trainers and participants are not getting through the lab because the QnA bot, which is at the end of the lab, was broken and I received no issues.  I would suggest pairing back the lab to the essential concepts not covered in another MCW lab.  Do we need bots and logic apps?
- Update the web app to use .NET Core or at least update to .NET Framework 4.7
- Update the NuGet packages to the latest versions.
- Update the LUIS authentication feature.  On August 31, 2020, your current authoring keys will be retired.
- Look into possibly using Azure Key Vault for connection strings.
- Remove passwords from the lab.
- Update lab to use separate Event Hub policies for Send and Receive, instead of Manage.
- Add a little more information regarding partitioning and the reason for using certain fields as keys.
- Add little more information why auto-inflate would be a good idea for scalability.
- Allow sentiment score to be retrievable and searchable.

**Comment by timahenning** [comment]
- Updated the solution application from .NET Framework to using all .NET Core 3.1.  Updated the application to use ASPT .NET Core SignalR instead of Web Sockets.  Updated all of the dependencies to use the latest frameworks.
- The LUIS portion of the application changed dramatically.  I added a diagram describing the flow for creating a LUIS model.  The response object changed and I updated the classes in C# code.  I pointed out the machine learning features when you enter in new utterances and how ML predicts the entity.
